presentation/src/main/java/io/forus/me/android/presentation/view/screens/main/MainActivity.kt

```diff
@@ -182,7 +182,7 @@ class MainActivity : BaseActivity() {
 
     private fun processInAppUpdateError(error: String) {
 
-        Toast.makeText(this@MainActivity, "initInAppUpdate error $error", Toast.LENGTH_LONG).show()
+        //Toast.makeText(this@MainActivity, "initInAppUpdate error $error", Toast.LENGTH_LONG).show()
```

Comment:
Could you add the word 'todo' when you comment things out? So you can search later to remove all todo's if needed